test_scripts/API/VehicleData/StabilityControlsStatus/019_UnsubscribeVehicleData_StabilityControlsStatus_disallowed_by_policy.lua

```diff
@@ -0,0 +1,36 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0253-New-vehicle-data-StabilityControlsStatus.md
+--
+-- Description: UnsubscribeVehicleData RPC with the only parameter stabilityControlsStatus, which is NOT allowed by Policies
+--
+-- Precondition:
+-- 1) SDL and HMI are started
+-- 2) App is registered
+-- 3) PTU is successfully performed, stabilityControlsStatus param is NOT allowed by Policies
+-- 4) App is activated
+--
+-- Steps:
+-- 1) App sends UnsubscribeVehicleData (with stabilityControlsStatus = true) request to SDL
+-- SDL does:
+--  - not sends VehicleInfo.UnsubscribeVehicleData request to HMI
+--  - send UnsubscribeVehicleData response with (success: false, resultCode: "DISALLOWED",
+--    stabilityControlsStatus = { resultCode = "DISALLOWED", dataType = "VEHICLEDATA_STABILITYCONTROLSSTATUS"})
+
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('test_scripts/API/VehicleData/commonVehicleData')
+
+--[[ Scenario ]]
+common.Title("Preconditions")
+common.Step("Clean environment", common.preconditions)
+common.Step("Start SDL, init HMI, connect default mobile", common.start)
+common.Step("Register App", common.registerApp)
+common.Step("PTU", common.policyTableUpdate, { common.ptUpdateMin })
```

Comment:
@LitvinenkoIra  Performed PTU disallows stabilityControlsStatus parameter not only for tested in this script RPC (SubscribeVehicleData). So the script can miss possible class of defects when policy works incorrect for new parameter.
I would propose to disable only new parameter for only tested RPC (stabilityControlsStatus parameter for SubscribeVehicleData)

This comment relates next script too.